api/candlepin-api-spec.yaml

```diff
@@ -734,6 +734,11 @@ paths:
             items:
               type: string
               format: key:value
+        - name: environmentId
```

Comment:
I think we use snake case instead of camel case for API parameters, so lets change this to environment_id

Reply:
done